src/main/java/supersymmetry/api/recipe/properties/SinterProperty.java

```diff
@@ -0,0 +1,28 @@
+package supersymmetry.api.recipe.properties;
+
+import gregtech.api.recipes.recipeproperties.RecipeProperty;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.resources.I18n;
+import org.jetbrains.annotations.NotNull;
+
+public class SinterProperty extends RecipeProperty<Boolean> {
+
+    public static final String KEY = "plasma_enabled";
+    private static SinterProperty INSTANCE;
+
+    private SinterProperty() {
+        super(KEY, Boolean.class);
+    }
+
+    @Override
+    public void drawInfo(@NotNull Minecraft minecraft, int x, int y, int color, Object value) {
+        minecraft.fontRenderer.drawString(I18n.format("gregtech.recipe.plasma_requirement", castValue(value)), x, y, color);
```

Comment:
You may want to change this to `if (castValue(value)) draw()`. That way the line would only show up if it is required.